### PR TITLE
License the software under Apache 2.0

### PR DESCRIPTION
**Pull request by akoehn**
Licensing this software under the Apache 2.0 license was agreed upon
by all contributors in GitHub issue #190.

### CODE REVIEW

File: bin/anthology/latexcodec.py [hunk 1]
```diff
@@ -1,3 +1,19 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Marcel Bollmann <marcel@bollmann.me>
```
Comment (mbollmann) [inline bin/anthology/latexcodec.py]: This file's not mine. It was originally written by David Eppstein, and the main source appears to be [this code recipe on ActiveState](http://code.activestate.com/recipes/252124-latex-codec/), where it claims to be PSFL licensed. Furthermore I think our version of this file has been heavily modified by @mjpost, @danielgildea, and @davidweichiang (but most notably -- not me :)).

Comment (mjpost) [reply bin/anthology/latexcodec.py]: Probably we can just swap that license in the header here, then?

https://docs.python.org/3/license.html

Comment (akoehn) [reply bin/anthology/latexcodec.py]: I'll update the pull request on Monda or Tuesday.

**Comment by akoehn** [comment]
Ok, I updated the PR via force-push as I don't want to have incorrect license headers in the history.

**Comment by mjpost** [comment]
One further other thought here—should we also ask authors to turn over copyright? The license means that we don't have to worry, but it can get kind of strange to see copyright for a file owned by a single person after many years of many people contributing to it. An alternative would be to ask people to turn copyright over to the ACL (which is what the ACL does for scientific papers).

**Comment by akoehn** [comment]
The file is not owned by a single person if it has multiple contributors. I added all the authors who added functionality to the header; contributors should do that as well if they add to the code.  It is an unfortunate fact that in a lot of cases this does not happen, but a) there is still the option of `git shortlog -n -s` to get the list of contributors and b) it is their own decision.  I don't see relevant problems with this approach

On the other hand, assigning copyright to the ACL *does* have problems.  Some people might not be allowed to assign copyright due to their employer, all contributors have to sign an agreement befor contributing (raising the bar for drive-by contributions), someone has to keep track of all agreements, etc. pp.

**Comment by akoehn** [comment]
Most Free Software projects work without copyright assignment (linux being the most prominent one) and the only two benefits are: 
 - ability to re-license the code (not needed in this case as Apache 2.0 is already extremely permissive) and
 - ability do defend copyright in court in case of infringement (hard to see how this could ever be needed & the current contributors could not do that themselves)

Therefore, please let this project work without copyright assignment.  

P.S.: If you ask why I did not add myself to the copyright headers:  There was no intellectual work in my changes worthy of copyright.

**Comment by mjpost** [comment]
Okay, this sounds good to me. I just wanted to make sure our bases were covered, and this response makes sense to me.

**Comment by mbollmann** [comment]
Thanks @akoehn!

**Comment by mjpost** [comment]
Yes—pardon my lapse, here—thank you very much!

**Pull request merged**